### PR TITLE
Re-rename package to @expensify/react-native

### PR DESCRIPTION
**Pull request by roryabraham**
Renames package to `@expensify/react-native` and bumps the version to `0.70.1`. This is the branch I published `0.70.1` from.

## Changelog

[CATEGORY] [TYPE] - Message

### CODE REVIEW

File: sdks/hermes-engine/hermes-engine.podspec [hunk 1]
```diff
@@ -12,7 +12,7 @@ import_hermesc_file=File.join(__dir__, "..", "hermesc", "osx-bin", "ImportHermes
 # package.json
 package_file = File.join(__dir__, "..", "..", "package.json")
 package = JSON.parse(File.read(package_file))
-version = package['version']
+version = '0.70.0'
```
Comment (AndrewGable) [inline sdks/hermes-engine/hermes-engine.podspec:15]: Should this be 70.1?

Comment (roryabraham) [reply sdks/hermes-engine/hermes-engine.podspec:15]: Nope - we're using the Hermesc build artifact from RN 0.70.0, but our package version is 0.70.1

**Comment by AndrewGable** [comment]
https://github.com/Expensify/react-native/pull/31#issuecomment-1239153301 is the only thing blocking

**Comment by AndrewGable** [comment]
Do we still want to do this?

**Comment by roryabraham** [comment]
The branch we publish from already has this change. Main is (perhaps confusingly) prettymuch unused in this repo

**Comment by AndrewGable** [comment]
So should we merge or close this?

**Pull request closed without merging**